crates/symbolicator-js/src/api_lookup.rs

```diff
@@ -70,13 +70,15 @@ pub struct SentryLookupApi {
     runtime: tokio::runtime::Handle,
     js_cache: SentryJsCache,
     timeouts: DownloadTimeouts,
+    propagate_traces: bool,
 }
 
 impl fmt::Debug for SentryLookupApi {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("SentryDownloader")
+        f.debug_struct("SentryLookupApi")
```

Comment:
What happened there? 😅

Reply:
I just found this and fixed it in drive-by. Apparently the debug implementation was copied from `SentryDownloader` and the rename was overlooked.